core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -173,7 +173,8 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
               // from the buffer
               val (_, newBuffer) = runBuffer.dequeue
               runBuffer = newBuffer
-              runBuffer.dequeueOption.foreach { case (run, _) => self ! run }
```

Comment:
do you mind fixing this comment on line 170
            // Remove the action that get's executed now from the buffer and execute the next one afterwards.
to
            // Remove the action that was just executed from the buffer and execute the next one in the queue.

Reply:
github is not showing this update 🤷‍♂